src/backend/templates/views/account/register.pug

```diff
@@ -13,6 +13,10 @@ block content
                 br
                 +flash-messages(flash)
 
+                hr
+                h3 Lost account / returning player
+                p If you previously had a FAF account you no longer can get access to, please contact the moderation team via discord or forums, or see if you can reset your password using the Steam API. Note that creating and operating multiple accounts is prohibited, and creating a secondary account could lead to moderation action.
```

Comment:
Slight rewording. What do you think?

```suggestion
                p If you previously had a FAF account you can no longer access, please contact the moderation team via Discord or the forums, or see if you can reset your password using the Steam API. Note that creating and operating multiple accounts is prohibited, and creating a secondary account could lead to moderation action.
```

Reply:
Text with formatting and links removed:

If you have a FAF account you can no longer access, please contact the moderation team via Discord or the Forums, or attempt to reset your password on Steam
Creating and operating multiple accounts is prohibited, and creating a secondary account will lead to moderation action.

<img width="1531" alt="Screenshot 2024-01-08 at 07 47 33" src="https://github.com/FAForever/website/assets/50498548/f5b7a3a8-c66c-4d86-9279-e7b454c96062">

Reply:
@pumbas600 
Added your suggestions and a few other tweaks to make it a bit less verbose.
Not 100% about including links for discord/steam since if those change then it will require someone to go in and edit them

Reply:
I would probably put the reset your password using steam first since that normally always works and is something the user can do on their own.

Reply:
<img width="1727" alt="Screenshot 2024-01-08 at 10 48 04" src="https://github.com/FAForever/website/assets/50498548/8a92dd59-ea9a-498d-84ef-ec06d2651f70">

Reply:
> including links for discord/steam since if those change then it will require someone to go in and edit them

I've not properly looked through the pug files, but I imagine we're also using these links in the top right of the website? I believe you can create variables for these links and then use the variables in the pug files, rather than hardcoding it directly. That would mean if they needed to be changed, it could be done in one place :)

Reply:
ping me if you need some input/merge, nice to see some work being done :)

Reply:
"could lead to moderation action"? That sounds like we will only maybe enforce this. I would suggest either rewording it to "will lead to moderator action" or shortening the whole sentence to "Note that creating and operating multiple accounts is prohibited." The second half is a bit redundant

Reply:
> "could lead to moderation action"? That sounds like we will only maybe enforce this. I would suggest either rewording it to "will lead to moderator action" or shortening the whole sentence to "Note that creating and operating multiple accounts is prohibited." The second half is a bit redundant

Didn't include the second line in my last 
`Creating and operating multiple accounts is prohibited, and creating a secondary account will lead to moderation action.`

I see the redundancy however and agree shortening is fine here.